### PR TITLE
GOPROXY docs improved

### PR DESCRIPTION
**Pull request by Sammers21**
Related to #361.

### CODE REVIEW

File: examples/go/README.md [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
   type: go
-  path: go
   storage:
     type: fs
-    path: /tmp/artipie/data/go
+    path: /var/artipie/data
 ```
 
-To use it for installing packages add it to `GOPROXY` environment variable:
+After creating the configuration file below, the configured GOPROXY is ready for use. First, you have to declare the following environment variables:
```
Comment (olenagerasimova) [inline examples/go/README.md:18]: @Sammers21 `First` seems odd as there is no `second` or something similar

File: examples/go/README.md [hunk 2]
```diff
@@ -1,43 +1,47 @@
...
-`example.com/foo/bar` versions `0.0.1` and `0.0.2`:
+Or, in case you want to stay connected with central proxy.golang.org, declare it in
+the following way:
 
+To use it for installing packages add it to `GOPROXY` environment variable:
+
+```bash
+export GO111MODULE=on
+export GOPROXY=http://localhost:8080/my-go
+export GOPROXY="http://localhost:8080/my-go,https://proxy.golang.org,direct"
```
Comment (olenagerasimova) [inline examples/go/README.md:33]: @Sammers21 I do not think we need two configuration examples, `go` developers should know that `GOPROXY` excepts list and it's highly unlikely that someone do not need to stay connected to central

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Sammers21 added comments, take a look please

**Review by olenagerasimova — APPROVED** [review]
@Sammers21 thanks
@olegmoz thanks, looks good to merge

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#591` is not assigned, can't get performer

**Pull request merged**